Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/DonorModule.kt

```diff
@@ -0,0 +1,32 @@
+package de.rki.coronawarnapp.datadonation.analytics.modules
+
+interface DonorModule {
```

Comment:
I very much like that approach: clean, good separation of concerns, and easily extendable